ClientApp/viking-vault/src/components/styles.css

```diff
@@ -155,13 +138,50 @@
     font-weight: lighter;
 }
 
+@keyframes fadein {
+    from {
+        opacity:0;
+    }
+    to {
+        opacity:1;
+    }
+}
+@-moz-keyframes fadein { 
+    from {
+        opacity:0;
+    }
+    to {
+        opacity:1;
+    }
+}
+@-webkit-keyframes fadein { 
+    from {
+        opacity:0;
+    }
+    to {
+        opacity:1;
+    }
+}
+@-o-keyframes fadein { 
+    from {
+        opacity:0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
 .dashboard-hide{
     display:none;
 }
 
 .dashboard-show{
     color: #09424d;
     font-weight: bold;
+    animation: fadein 2s;
+    -moz-animation: fadein 2s;
```

Comment:
The build system the `create-react-app` comes with automatically adds prefixes to the CSS